user_pinger.py

```diff
@@ -227,7 +227,11 @@ def handle_ping(self, group: str, comment: praw.models.Comment) -> None:
         self.logger.debug("Checking if author is in group or group is public")
         if not (self.in_group(comment.author, users) or self.public_group(group) or self.is_moderator(comment.author)):
             self.logger.warning("Non-member %s tried to ping \"%s\" group", comment.author, group)
-            self._send_error_pm(f"Cannot ping Group {group}", [f"You need to be a member of {group} to ping it", self._command_link(f"Click here, then click \"send\" to join {group}", f"Join {group}", "addtogroup", f"{group}") ], comment.author)
+            self._send_error_pm(f"Cannot ping Group {group}", [f"You need to be a member of {group} to ping it",
+                                                               self._command_link(
```

Comment:
Indentation

Reply:
oh wait I see what it's doing.

If you're gonna use IntelliJ autoformatting, I'd prefer continuation indents with one argument per line: 
```
method(
  a, 
  b, 
  c
); 
```
rather than the chop-down occurring here. IntelliJ has configuration for this.